src/consts.rs

```diff
@@ -25,7 +25,7 @@ pub const DEFAULT_ENVIRONMENT_NAME: &str = "default";
 /// The default channels to use for a new project.
 pub const DEFAULT_CHANNELS: &[&str] = &["conda-forge"];
 pub const DEFAULT_FEATURE_NAME: &str = DEFAULT_ENVIRONMENT_NAME;
-pub const ENVIRONMENT_FILE_NAME: &str = "pixi_environment.toml";
+pub const ENVIRONMENT_FILE_NAME: &str = "pixi_environment.json";
```

Comment:
Since you place this in the `conda-meta` folder doesnt this cause issues?

Reply:
And could we call it just `pixi.json` perhaps?

Reply:
I don't place it in conda-meta, should it?

Reply:
We had an issue with the prefix file that it was overwritten right? Thats why we placed it in the conda-meta dir.